### PR TITLE
Representing current Treat-AD with latest template

### PR DESCRIPTION
**Pull request by mfazza**
## What's Inside

Representation of current TREAT-AD Internal Sharing Site using the latest long form YAML.

### CODE REVIEW

File: templates/treat_ad_long.yaml [hunk 1]
```diff
@@ -0,0 +1,908 @@
+- name: TREAT-AD Internal Sharing Site
+  type: Project
+  acl:
+  - principal_id: 3425650
```
Comment (thomasyu888) [inline templates/treat_ad_long.yaml]: nit (minor): Did you want to leave comments for who these principals are to improve readability for yourself?  
```
  - principal_id: 3425650 # user 1
```

File: templates/treat_ad_long.yaml [hunk 2]
```diff
@@ -0,0 +1,908 @@
...
+          type: Folder
+          children:
+          - name: PRDX1
+            type: Folder
+          - name: PRDX6
+            type: Folder
+          - name: EPHX2
+            type: Folder
+          - name: CAPN2
+            type: Folder
```
Comment (thomasyu888) [inline templates/treat_ad_long.yaml]:
```suggestion
            type: Folder

```

**Comment by thomasyu888** [comment]
LGTM! 🚀

**Review by thomasyu888 — APPROVED** [review]

File: templates/treat_ad_long.yaml (hunk 1 above)
Comment (mfazza) [reply templates/treat_ad_long.yaml]: I love that idea.  I have no idea who those are by looking at the number.  Thanks for the feedback, Tom!

**Pull request merged**